### PR TITLE
New messages are added in Spanish

### LINKED ISSUE #19959: Add missing Spanish translations for login
### Discussed in https://github.com/keycloak/keycloak/discussions/19945

<div type='discussions-op-text'>

<sup>Originally posted by **jorozcof** April 25, 2023</sup>
The next message is a "kc-error-message". It happens when I try to link an email account that does not exist in the registered users. The flow that I am using is "first broker login"

Is there a possibility to translate the message? In the realm settings I have Default locale in Spanish but the internationalization is not applicable here.

image
</div>

### PR DESCRIPTION
**Pull request by jorozcof**
Closes #19959

### CODE REVIEW

**Review by rmartinc — CHANGES_REQUESTED** [review]
Thanks for the PR  @jorozcof!

Just some minor comments because the messages mix second and third person, better use third person in all of them.

File: themes/src/main/resources-community/theme/base/login/messages/messages_es.properties [hunk 1]
```diff
@@ -192,10 +195,25 @@ identityProviderNotFoundMessage=No se encontr\u00F3 un proveedor de identidad.
...
 
 backToApplication=&laquo; Volver a la aplicaci\u00F3n
 missingParameterMessage=Par\u00E1metros que faltan: {0}
 clientNotFoundMessage=Cliente no encontrado
 invalidParameterMessage=Par\u00E1metro no v\u00E1lido: {0}
-alreadyLoggedIn=You are already logged in.
+alreadyLoggedIn=Ya se ha autentificado.
 
+# Identity provider
+identity-provider-redirector=Con\u00E9ctese con otro proveedor de identidad
```
[the remark this answers is not in the document]
Comment (jorozcof) [reply themes/src/main/resources-community/theme/base/login/messages/messages_es.properties:208]: Yes, It sounds weird. I think is better `Inicie sesi\u00F3n con otro proveedor de identidad`. What do you thing @rmartinc ?

[the remark this answers is not in the document]
Comment (rmartinc) [reply themes/src/main/resources-community/theme/base/login/messages/messages_es.properties:208]: Yep, it sounds good to me.

**Review by rmartinc — APPROVED** [review]
Thanks @jorozcof! LGTM.

**Comment by rmartinc** [comment]
Yes, no problem, you can add them in this PR. I'm going to set it to `request changes` to not merge it by mistake.

**Review by rmartinc — CHANGES_REQUESTED** [review]
@jorozcof wants to add some more missing messages in Spanish.

**Comment by jorozcof** [comment]
> @jorozcof wants to add some more missing messages in Spanish.

@rmartinc Yes, I added new messages in Spanish. If you like, you can review in my last commit.

**Review by rmartinc — APPROVED** [review]
Thanks @jorozcof ! LGTM!

**Review by mposolda — APPROVED** [review]
@jorozcof @rmartinc Thanks for the PR and review! Merging based on @rmartinc review

**Pull request merged**